lib/input/check_headers.py

```diff
@@ -125,14 +125,15 @@ def check_validity(self, check_unavailable_uid='False', json_header_filename='he
 		return self.last_uid_read
 
 
-	def remove_unwanted_headers(self, to_remove=unwanted_uid, json_header_filename='headers.json'):
+	def remove_unwanted_headers(self, to_remove=unwanted_uid, json_header_filename="headers.json", output_filename="headers.json"):

```

Comment:
In the previous signature `json_header_filename` was being used to read input and update the output. We can use the same model to simplify the number of parameters. Also `json_headers` is a better name than `json_header_filename`. Please consider making the change for other functions as well.

Reply:
@prasadtalasila The issue with using the same parameter for input and updated output is that the file changes after every execution and hence gives errors with testing.

Reply:
Makes sense. Please go ahead with your convention.